master: Update pkg/testutils/release/cockroach_releases.yaml

cockroach-teamcity opened the pull request:
Update pkg/testutils/release/cockroach_releases.yaml with recent values.

Epic: None
Release note: None
Release justification: test-only updates

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/130629)
<!-- Reviewable:end -->

rail commented:
bors r+

rail commented:
bors retry

bitomukesh commented:
<div>
<h2>Changelist by Bito</h2>
    <p>This pull request implements the following key changes.</p>

<div>
<table width="100%">
<tbody>
  <tr>
    <th>Key Change</th>
    <th>Files Impacted</th>
  </tr>

<tr>
<td width="30%">
    <b>Other Improvements</b> - Update CockroachDB Release Information
</td>
<td width="70%">


<div>

-&nbsp;[`cockroach_releases.yaml`](https://github.com/cockroachdb/cockroach/pull/130629/files#diff-b57efe28a222bfc7db781c677207e3cde46e2cd67a35db1bae3075ea6fa3bdda) - Updated latest versions and added withdrawn version
</div>
</td>
</tr>

</tbody>
</table>
</div>
</div>

cockroach-teamcity committed:
master: Update pkg/testutils/release/cockroach_releases.yaml

Update pkg/testutils/release/cockroach_releases.yaml with recent values.

Epic: None
Release note: None
Release justification: test-only updates

rail commented:
bors r+
